### DISCUSSION TITLE
Are there projects?

### DISCUSSION THREAD

**Discussion by glow12121** [helix-editor/helix#5829] [Q&A]
Is it possible to save a project in Hx? I merely need 3 files to be kept opened in a certain layout (with splits) when I start the editor next time.

Or, maybe this could be done through "session" functionality.

Didn't find anything in the docs.

**Comment by the-mikedavis** [comment]
Remembering which files were opened and split configuration would need #401.

If you're only opening the same files and only use one kind of split, you could use the `--vsplit` or 
`--hsplit` CLI switch, for example `hx --vsplit a.txt b.txt c.txt` will open `a.txt`, `b.txt` and `c.txt` in three vertical splits.

**Reply by glow12121** [reply]
Thanks for letting me know, that's useful.
